### DISCUSSION TITLE
when putting ' in filter, gives error.

### DISCUSSION THREAD

**Discussion by kenyonbowers** [pocketbase/pocketbase#1056] [Q&A] [answered]
So I have this filter,
params = { filter: `(firstName~\`${token}\`||lastName~\`${token}\`)` }
And I don't know why when the token variable has an ' in it, it gives an error saying invalid filter parameters. Why is that?

**Comment by ganigeorgiev** [comment, marked as answer]
The js variable must be wrapped with a single or double quotes, othewise the filter will search for a field/identifier with your variable value. Or in other words, your filter should be something like:

```js
params = { filter: `firstName ~ "${token}" || lastName ~ "${token}"` }
```

**Reply by kenyonbowers** [reply]
Oh I see. Thank you.

**Reply by Eggtron** [reply]
Thanks, I searched for that info a few hours :)
